public/theme/boltcms-2021/listing.twig

```diff
@@ -0,0 +1,30 @@
+{% extends 'partials/_master.twig' %}
+
+{% block main_container %}
+
+    <div class="content-wrapper">
+        <div class="current-version">
+            <p>Latest version of Bolt CMS: 4.1.13 <a href="https://github.com/bolt/core/releases/latest">Download this
```

Comment:
Looks very cool! We'd need to make sure the latest version is automatically updated, I'm not sure if it is trivial to do.

Let's revisit together :-)

Reply:
Check